plugins/modules/proxysql_mysql_users.py

```diff
@@ -177,6 +190,21 @@ def load_config_to_runtime(cursor):
     return True
 
 
+def mysql_native_password(cleartext_password):
+    mysql_native_encrypted_password = "*" + sha1(sha1(cleartext_password.encode('utf-8')).digest()).hexdigest().upper()
```

Comment:
I think we can import `to_bytes` from `from ansible.module_utils._text` (alongside `to_native`), and use that instead of .encode here.

Reply:
Ahh, yes, I wasn't aware of `to_bytes()`. Done.... however, I realise I've messed it up, I'll wait till CI finishes and fails, then fix it.

Reply:
Now fixed.